python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -5008,30 +5008,33 @@ def test_cov_nans():
         operator.truediv,
         operator.mod,
         operator.pow,
-        operator.eq,
-        operator.lt,
-        operator.le,
-        operator.gt,
-        operator.ge,
-        operator.ne,
+        # TODO: Remove XFAILs after PR is merged
+        pytest.param(operator.eq, marks=pytest.mark.xfail()),
```

Comment:
Can we remove these xfails? If not should we put in the url to PR here?

Reply:
this was dependent on PR https://github.com/rapidsai/cudf/pull/7490, and was removed 👍